packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.component.tsx

```diff
@@ -116,7 +116,7 @@ const RenderBiometrics: React.FC<RenderBiometricsProps> = ({
                   </TableHead>
                   <TableBody>
                     {rows.map((row) => (
-                      <TableRow key={row.id}>
+                      <TableRow key={row.id} style={{ borderBottom: `2px solid #e0e0e0` }}>
```

Comment:
Is this the general table design? Because if so, I'd say let's override the style in general (in esm-core -> esm-styleguide).

Reply:
@FlorianRappl I think this should a general table design and not Vitals specific